code/src/index.css

```diff
@@ -11,3 +17,329 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.loader{
+  position: fixed;
+  justify-content: center;
+  top: 45%;
+  left: 45%;
+}
+
+.popcorn-spinner{
+  animation-name: spin;
+  animation-duration: 5000ms;
+  animation-iteration-count: infinite;
+  animation-timing-function: linear; 
+}
+@keyframes spin {
+  from {
+      transform:rotate(0deg);
+  }
+  to {
+      transform:rotate(360deg);
+  }
+}
+
+main {
+  max-width: auto;
+  margin: 0 auto;
+  width: 100%;
```

Comment:
There's something that makes the width greater than screen sizes under 375px. So the page is scrollable horizontally. :(